docs/setup/settings.asciidoc

```diff
@@ -231,6 +231,16 @@ This value must be a whole number greater than zero. *Default: `"1000"`*
 suggestions. This value must be a whole number greater than zero.
 *Default: `"100000"`*
 
+|===
+
+[NOTE]
+============
+Logging settings can be reloaded issuing a `SIGHUP` signal to Kibana.
```

Comment:
To reload the logging settings, send a SIGHUP signal to {kib}.

Reply:
I did some digging, and it looks like we can use SIGHUP without any annotations. Also, the documentation says that you send a SIGHUP signal instead of issues, so I added that too.